assets/prototype/domain/eventEditor/containers/datesListData/index.js

```diff
@@ -0,0 +1,81 @@
+import { createContext, useReducer, useEffect, useState } from '@wordpress/element';
+import gql from 'graphql-tag';
+import { useQuery } from '@apollo/react-hooks';
+
+const GET_DOGS = gql`
+	query getEventData($eventId: Number!) {
+		eventBy(eventId: $eventId) {
+			name
+		}
+	}
+`;
+
+const DEFAULT_STATE = {
+	datetimes: [],
+};
+
+const reducer = ( state, action ) => {
+	switch ( action.type ) {
+		case 'setDatetimes':
+			console.log( 'action.payload', action.payload );
+			return { datetimes: action.payload };
+		case 'add':
+			return { todoList: [ ...state.todoList, action.payload ] };
+	}
+};
```

Comment:
I don't think we need to re-invent the wheel when the data is already in cache.

Reply:
This is outdated, has been cleaned up here: https://github.com/eventespresso/event-espresso-core/pull/1876/commits/ed2145ef7002d1660cb2d3e3a893dce1e5c74b40

Reply:
We don't need to use `useReducer` here.